wsgi.py

```diff
@@ -4,10 +4,14 @@
 import os
 
 
+__flask_app__ = create_ui_web_app()
+
+
 def application(environ, start_response):
     """WSGI application factory."""
     for key, value in environ.items():
-        if type(value) is str:
-            os.environ[key] = value
-    app = create_ui_web_app()
-    return app(environ, start_response)
+        if key == 'SERVER_NAME':
```

Comment:
A brief comment for this exception would be helpful here.

Reply:
Added in bf05185